### ISSUE TITLE
Streaming Convnets default recipe error

### ISSUE THREAD

**Issue by kksiddharth** [facebookresearch/wav2letter#589]
Hi there,
  I was trying out the default recipe for streaming convnets for libri speech, and I have encountered the following error regarding the arch file, the stderr logs are

I0329 07:51:35.575857 29457 Train.cpp:58] Reading flags from file ./train_am_500ms_future_context.cfg
[[21735,1],0]: A high-performance Open MPI point-to-point messaging module
was unable to find any relevant network interfaces:
Module: OpenFabrics (openib)
  Host: speech-dl-01
Another transport will be used instead, although this may result in
lower performance.
NOTE: You can disable this warning by setting the MCA parameter
btl_base_warn_component_unused to 0.
Initialized NCCL 2.4.8 successfully!
I0329 07:51:36.112752 29457 Train.cpp:137] Gflags after parsing 
--flagfile=; --fromenv=; --tryfromenv=; --undefok=; --tab_completion_columns=80; --tab_completion_word=; --help=false; --helpfull=false; --helpmatch=; --helpon=; --helppackage=false; --helpshort=false; --helpxml=false; --version=false; --adambeta1=0.90000000000000002; --adambeta2=0.999; --am=; --arch=am_500ms_future_context.arch; --archdir=/home/developer2/wav2letter/wav2letter/recipes/models/streaming_convnets/librispeech; --attention=content; --attentionthreshold=0; --attnWindow=no; --attnconvchannel=0; --attnconvkernel=0; --attndim=0; --batchsize=8; --beamsize=2500; --beamsizetoken=250000; --beamthreshold=25; --blobdata=false; --channels=1; --criterion=ctc; --critoptim=sgd; --datadir=; --dataorder=input; --decoderattnround=1; --decoderdropout=0; --decoderrnnlayer=1; --decodertype=wrd; --devwin=0; --emission_dir=; --enable_distributed=true; --encoderdim=0; --eosscore=0; --eostoken=false; --everstoredb=false; --fftcachesize=1; --filterbanks=80; --flagsfile=./train_am_500ms_future_context.cfg; --framesizems=25; --framestridems=10; --gamma=1; --gumbeltemperature=1; --input=flac; --inputbinsize=100; --inputfeeding=false; --iter=1000000; --itersave=false; --labelsmooth=0; --leftWindowSize=50; --lexicon=/home/developer2/wav2letter/wav2letter/recipes/models/streaming_convnets/librispeech/am/librispeech-train+dev-unigram-10000-nbest10.lexicon; --linlr=-1; --linlrcrit=-1; --linseg=0; --lm=; --lm_memory=5000; --lm_vocab=; --lmtype=kenlm; --lmweight=0; --localnrmlleftctx=300; --localnrmlrightctx=0; --logadd=false; --lr=0.40000000000000002; --lrcosine=false; --lrcrit=0; --maxdecoderoutputlen=200; --maxgradnorm=0.5; --maxisz=33000; --maxload=-1; --maxrate=10; --maxsil=50; --maxtsz=9223372036854775807; --maxword=-1; --melfloor=1; --memstepsize=10485760; --mfcc=false; --mfcccoeffs=13; --mfsc=true; --minisz=200; --minrate=3; --minsil=0; --mintsz=2; --momentum=0; --netoptim=sgd; --noresample=false; --nthread=6; --nthread_decoder=1; --numattnhead=8; --onorm=target; --optimepsilon=1e-08; --optimrho=0.90000000000000002; --outputbinsize=5; --pctteacherforcing=100; --pcttraineval=1; --pow=false; --pretrainWindow=0; --replabel=0; --reportiters=1000; --rightWindowSize=50; --rndv_filepath=; --rundir=/home/developer2/wav2letter/wav2letter; --runname=inference_2019; --samplerate=16000; --sampletarget=0; --samplingstrategy=rand; --sclite=; --seed=0; --show=false; --showletters=false; --silscore=0; --smearing=none; --smoothingtemperature=1; --softwoffset=10; --softwrate=5; --softwstd=5; --sqnorm=true; --stepsize=1000000; --surround=; --tag=; --target=tkn; --test=; --tokens=librispeech-train-all-unigram-10000.tokens; --tokensdir=/home/developer2/wav2letter/wav2letter/recipes/models/streaming_convnets/librispeech/am; --train=/home/developer2/w2l/lists/lists/train-clean-100.lst; --trainWithWindow=false; --transdiag=0; --unkscore=-inf; --use_memcache=false; --uselexicon=true; --usewordpiece=true; --valid=/home/developer2/w2l/lists/lists/dev-clean.lst; --weightdecay=0; --wordscore=0; --wordseparator=_; --world_rank=0; --world_size=1; --alsologtoemail=; --alsologtostderr=false; --colorlogtostderr=false; --drop_log_memory=true; --log_backtrace_at=; --log_dir=; --log_link=; --log_prefix=true; --logbuflevel=0; --logbufsecs=30; --logemaillevel=999; --logfile_mode=436; --logmailer=/bin/mail; --logtostderr=true; --max_log_size=1800; --minloglevel=0; --stderrthreshold=2; --stop_logging_if_full_disk=false; --symbolize_stacktrace=true; --v=0; --vmodule=; 
I0329 07:51:36.112826 29457 Train.cpp:138] Experiment path: /home/developer2/wav2letter/wav2letter/inference_2019
I0329 07:51:36.112854 29457 Train.cpp:139] Experiment runidx: 1
I0329 07:51:36.116801 29457 Train.cpp:183] Number of classes (network): 9998
I0329 07:51:36.364280 29457 Train.cpp:190] Number of words: 33758
I0329 07:51:36.384649 29457 Train.cpp:204] Loading architecture file from /home/developer2/wav2letter/wav2letter/recipes/models/streaming_convnets/librispeech/am_500ms_future_context.arch
F0329 07:51:36.682444 29457 W2lModule.cpp:127] Failed parsing - TDS 15 9 80 0.1 0 1 0
*** Check failure stack trace: ***
    @     0x7f264746a0cd  google::LogMessage::Fail()
    @     0x7f264746bf33  google::LogMessage::SendToLog()
    @     0x7f2647469c28  google::LogMessage::Flush()
    @     0x7f264746c999  google::LogMessageFatal::~LogMessageFatal()
    @     0x55c977df99a7  (anonymous namespace)::parseLines()
    @     0x55c977dfc030  w2l::createW2lSeqModule()
    @     0x55c977bed155  main
    @     0x7f2646520b97  __libc_start_main
    @     0x55c977c415ca  _start
Aborted (core dumped)

   I was searching about this bug and found a reply to a similar issue in 
https://github.com/facebookresearch/wav2letter/issues/563#issuecomment-601772239
I am using the latest pull, anyhelp in understanding this bug or fixing it will help
Thanks

**Comment by vineelpratap** [comment]
Hi,
 I tested again and it works for me. It seems to me you are not using the latest version.

**Comment by kksiddharth** [comment]
Hey, I think I am using the latest pull, git pull origin master, returns : 
From https://github.com/facebookresearch/wav2letter
 * branch            master     -> FETCH_HEAD
Already up to date.
Or is it some other branch that I am supposed to use. 
I built wav2letter in another dummy system, and this error doesn't seem to come up there. Is there some way for me to verify the out dated version, or do I have to rebuild a few binaries, if so can you let me know what would be the binaries that I may have to be rebuilt.
Thanks

**Comment by kksiddharth** [comment]
Hey, I rebuilt the Train binary and now the error seem to go, Thanks
